espnet2/enh/layers/tcn.py

```diff
@@ -94,9 +96,20 @@ def __init__(
         # [M, B, K] -> [M, C*N, K]
         mask_conv1x1 = nn.Conv1d(B, C * self.out_channel, 1, bias=False)
         # Put together (for compatibility with older versions)
-        self.network = nn.Sequential(
-            layer_norm, bottleneck_conv1x1, temporal_conv_net, mask_conv1x1
-        )
+        if pre_mask_nonlinear == "linear":
+            self.network = nn.Sequential(
```

Comment:
A minor question, if the `self.network` is used separately in forward (e.g. bottleneck, tcn, masknet called individually), what's the benefit of defining them in a Sequential(). I found it is a bit ambiguous in naming the subnet in forward().

Reply:
This is just for **back-compatibility** with Conv-TasNet. Because previously the TCN implementation also used `Sequential` for speech separation, which can be used as a whole. But for TD-SpeakerBeam, we cannot use the `Sequential` module directly because of the input/output mismatch between sub-modules.